BUILD.md

```diff
@@ -100,16 +110,18 @@ architecture:
 conan profile update settings.arch=x86_64 default
 ```
 
-If you have multiple compilers installed on your platform,
+**If you have multiple compilers** installed on your platform,
 then you'll need to make sure that Conan and CMake select the one you want to
 use.
-This setting will set the correct variables (`CMAKE_<LANG>_COMPILER`) in the
-generated CMake toolchain file:
-
+Update the `conf.tools.build:compiler_executables` setting in order to set the correct variables (`CMAKE_<LANG>_COMPILER`) in the
+generated CMake toolchain file.
+For example, on Ubuntu 20, you may have gcc at `/usr/bin/gcc` and g++ at `/usr/bin/g++`; if that is the case, you can select those compilers with:
 ```
-conan profile update 'conf.tools.build:compiler_executables={"c": "<path>", "cpp": "<path>"}' default
+conan profile update 'conf.tools.build:compiler_executables={"c": "/usr/bin/gcc", "cpp": "/usr/bin/g++"}' default
 ```
 
+Replace `/usr/bin/gcc` and `/usr/bin/g++` with paths to the desired compilers.
+
```

Comment:
Why does this example use real paths while the next example uses the `<path>` placeholder? Can we make this consistent?

Reply:
If people don't understand it one way, there's a nonzero chance they may at least understand it when it's provided in a different way.

Reply:
I think we need to first stop and examine why people do not understand it the way it is written. Or even _whether_ people do not understand it the way it is written. In fact, there _could_ be a zero chance that readers do not understand the different way, and a non-zero chance that _more_ readers are confused by the different way. @ckeshava can you explain which part of this instruction you didn't understand in your reading?

Reply:
I don't mind the changes. I think it is obvious either way to anyone with more than 30 min exposure to cmake.
There is also non zero chance that nothing good would come out of someone with no prior cmake and c++ experience trying to build rippled - they should just install the prebuilt packages 📦